pyiron/vasp/base.py

```diff
@@ -466,27 +466,37 @@ def collect_errors(self):
         """
         Collects errors from the VASP run
         """
-        num_eddrmm, snap = self._get_eddrmm_info()
+        num_eddrmm, snap_eddrmm = self._get_eddrmm_info()
 
-        warning_string = "EDDRMM warnings occured {} times, first in ionic step {}."
+        eddrmm_warning_string = "EDDRMM warnings occured {} times, first in ionic step {}."
+        zbrent_warning_string = "'ZBRENT: fatal error in bracketing' occured. Please check VASP manual for details."
         status_string = "Status is switched to 'warning'."
 
-        if snap is not None:
+        if snap_eddrmm is not None:
             if self.get_eddrmm_handling() == "ignore":
-                self._logger.warning(warning_string.format(num_eddrmm, snap))
+                self._logger.warning(eddrmm_warning_string.format(num_eddrmm, snap_eddrmm))
             elif self.get_eddrmm_handling() == "warn":
                 self.status.warning = True
-                self._logger.warning(warning_string.format(num_eddrmm, snap) + status_string)
+                self._logger.warning(eddrmm_warning_string.format(num_eddrmm, snap_eddrmm) + status_string)
             elif self.get_eddrmm_handling() == "restart":
                 self.status.warning = True
-                self._logger.warning(warning_string.format(num_eddrmm, snap) + status_string)
+                self._logger.warning(eddrmm_warning_string.format(num_eddrmm, snap_eddrmm) + status_string)
                 if not self.input.incar["ALGO"].lower() == "normal":
                     ham_new = self.copy_hamiltonian(self.name + "_normal")
                     ham_new.input.incar["ALGO"] = "Normal"
                     ham_new.set_eddrmm_handling()
                     ham_new.run()
                     self._logger.info("Job was restarted with 'ALGO' = 'Normal' to avoid EDDRMM warning.")
 
+        file_name = os.path.join(self.working_directory, "error.out")
+        if os.path.exists(file_name):
+            with open(file_name, "r") as f:
+                lines = f.readlines()
+                for l in lines:
+                    if "ZBRENT: fatal error in bracketing" in l:
+                        self.status.warning = True
```

Comment:
I think this is misleading. A warning status is assigned when the calculations have converged and some warnings are raised in between. But in your case, the calculation has actually crashed when the ZBRENT error is raised and the outputs are actually not written. So I think we should set the status to `aborted` or `not_converged`. What do you think?

Reply:
You are right. `aborted` is the only status to chose as the calculation is not complete. Remember the incomplete `vasprun.xml`.